DEVELOPER_GUIDE.md

```diff
@@ -153,35 +153,52 @@ extension_hw_greet:
     - "hw-user"
 ```
 
+### Initializing admin user with custom password
+
+From 2.11, users can set a custom admin password while running the demo setup. You can do this with one of the following 2 options: 
+1. By setting an environment variable. `export initialAdminPassword=<custom-admin-password>`
+2. By adding a single line as password in a file `initialAdminPassword.txt` under `$OPENSEARCH_HOME/config` directory.
+
+If neither of these are provided, the password defaults to `admin`. However, this will be removed in future releases, and a custom password will be required in order to use demo config.
```

Comment:
```suggestion

```

This would be a 3rd option, which we shouldn't allow, lets remove

Reply:
I have added this to account for: https://github.com/opensearch-project/security/pull/3490